src/main/java/io/debezium/connector/yugabytedb/transforms/StriimCompatible.java

```diff
@@ -0,0 +1,216 @@
+package io.debezium.connector.yugabytedb.transforms;
+
+import java.util.*;
+
+import io.debezium.connector.yugabytedb.transforms.SchemaUtil;
+import org.apache.kafka.common.config.ConfigDef;
+import org.apache.kafka.connect.connector.ConnectRecord;
+import org.apache.kafka.connect.data.Field;
+import org.apache.kafka.connect.data.Schema;
+import org.apache.kafka.connect.data.Schema.Type;
+import org.apache.kafka.connect.data.SchemaBuilder;
+import org.apache.kafka.connect.data.Struct;
+import org.apache.kafka.connect.transforms.Transformation;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.yb.util.Pair;
+
+/**
+ * A transformer to convert records in a format that is compatible with
+ * Striim's PostgreSQLReader format i.e. WAEvent.
+ * @param <R> Record
+ */
+public class StriimCompatible<R extends ConnectRecord<R>> implements Transformation<R> {
+    private static final Logger LOGGER = LoggerFactory.getLogger(StriimCompatible.class);
+
+    @Override
+    public R apply(final R record) {
+        if (record == null || (record.value() != null && !(record.value() instanceof Struct))) {
+            return record;
+        }
+
+        List<String> primaryKeys = getAllFieldsInOrder(record.keySchema());
```

Comment:
Nit: Is the variable name correct? Should it be called `columnNames` instead?

Reply:
Oh, I thought the `columnNames` in the key will be primary keys only. Isn't that the case @vrajat?

Reply:
I believe that is the case, the columns in keySchema are primary keys.